arith/src/serde.rs

```diff
@@ -56,6 +56,31 @@ field_serde_for_integer!(u64, 8);
 field_serde_for_integer!(usize, 8);
 field_serde_for_integer!(u8, 1);
 
+impl<V: FieldSerde> FieldSerde for Vec<V> {
```

Comment:
hmm, i guess this is implemented for `RawCommit`?

Reply:
I think it's good to have FieldSerde implemented for Vec, not only for RawCommit.